samples/shell/wordcount/wordcount.sh

```diff
@@ -1 +1,3 @@
-echo "$1" | wc -w
+#!/bin/sh
+
+echo `cat` | wc -w
```

Comment:
Should be able to be just `wc -n`, shouldn't it?

Reply:
I don't think `wc` supports `-n`: https://linux.die.net/man/1/wc

Reply:
Sorry, I meant just `wc -w`, ie no `echo`

Reply:
Yes, well spotted. Fixed.